ova/provision.sh

```diff
@@ -84,22 +157,21 @@ WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW.
 
 EOF
 
+# Clean cache
+yum clean all
 
-# Dependences
-yum install openssl -y
+# Remove data from /var/ossec/logs/
+cd / 
+find /var/ossec/logs/ -type f -exec sh -c ': > "$1"' - {} \;
 
-installPrerequisites
-addWazuhrepo
-addElasticRepo
-installWazuh
-installElasticsearch
-installFilebeat
-installKibana
-configWazuh
-checkInstallation
-cleanInstall
+# Remove data from /var/log/ files
+find /var/log/ -type f -exec sh -c ': > "$1"' - {} \;
 
-rm -rf /vagrant
+# Remove demo script
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/6fb50e3584df557334da525324a73a5c32ae0551